### ISSUE TITLE
Tilix guake mode bottom position leaves a slight open space from the bottom

### ISSUE THREAD

**Issue by SlyPex** [gnunn1/tilix#2090]
S i'm running this on an Arch linux, DE Gnome 41.4 WM Mutter.
i recently discovered guake mode in tilix , i did all the simple steps to set it up ( shortcut keybind + changing preferences )
I have the following preferences : 
![Screenshot from 2022-03-15 20-19-08](https://user-images.githubusercontent.com/59923212/158458020-7fdae7d3-e3ef-4c45-805b-c8ce8cc6b80c.jpg)

And after trying the guake mode of tilix i have this view out : 
![Screenshot from 2022-03-15 21-14-55](https://user-images.githubusercontent.com/59923212/158463895-ece41a5b-f739-44fa-8121-de786268eb8b.png)

As you can notice i have a slight open space from the bottom where i want it to covered by the guake mode of tilix.
idk if i have to provide more infos or not but i hope it's enough for this issue.

**Comment by slawtul** [comment]
Hi,
I checked it on my linux: EndeavourOS, DE: Xfce 4.16 and WM Xfwm4 and it works correctly when task bar panel is set horizontally or vertically. I built the Tilix and installed manually.
It seems that maybe you have some transparent panel set on the bottom of your screen. Could you please check? I do not know if new Gnome version has some panel with app icons.

![image](https://user-images.githubusercontent.com/4516171/161433078-359876c2-bfd2-411b-aa0c-2e6e46e86ca9.png)

![image](https://user-images.githubusercontent.com/4516171/161433114-c278c611-1ece-470d-a2fa-1783626af579.png)

**Comment by SlyPex** [comment]
Update :  i'm using Mutter-Rounded as a WM now but it's just like mutter no big difference.
Hey @slawtul thanks for your reply dude i really appreciate that.
About what you said on maybe there is a transparent panel or something , actually i don't know but i also don't think so cause if it was there all of the applications i use in here will have the same problem of being slightly away from the bottom.
But i did discover something which is when i try to maximize the width and the height i only can go up to 90 in the height as shown below :

is it the same for you ? like what the maximum value which you can set ?

![Screenshot from 2022-04-03 16-36-53](https://user-images.githubusercontent.com/59923212/161435774-002db8a1-b2cd-4e76-8162-638095c0110f.png)

Thank you so much.

**Comment by slawtul** [comment]
Hi, the same: max 90% of height

But I noticed on your screenshots that open space from the bottom is 37px of height like gnome top bar (where is calendar, sound icon, etc...)

![image](https://user-images.githubusercontent.com/4516171/161440645-61414804-2c9a-4f9f-85cd-987a4c8aa829.png)

**Comment by slawtul** [comment]
You could try to hide top bar and then test if this open space still shows  - maybe this extension will help [](https://extensions.gnome.org/extension/545/hide-top-bar/)

**Comment by slawtul** [comment]
Maybe you could also install base Adwaita theme to check if issue still exists?

**Comment by SlyPex** [comment]
That's true, i installed it and it works perfectly, but it destroys my theme xD that's sad i can't have both of them.

![Screenshot from 2022-04-03 19-11-21](https://user-images.githubusercontent.com/59923212/161442025-fb29f142-8252-4582-bdaf-ba8dd8cf0416.png)

@slawtul Thank you so much man i really appreciate your time.

i'll try the base Adwaita and check that

**Comment by slawtul** [comment]
You are welcome!
